src/ServiceControl.Plugin.Nsb6.CustomChecks.AcceptanceTests/ScenarioDescriptors/Persistence.cs

```diff
@@ -0,0 +1,92 @@
+﻿namespace NServiceBus.AcceptanceTests.ScenarioDescriptors
```

Comment:
Do we need all this given that we won't change persistence? (same for serializers etc etc)

Can we go with a "hardcoded" default server instead?